doc/endpoints/track.md

```diff
@@ -170,7 +184,27 @@ A promise that resolves with an object, containing audio analysis information fo
 
 ```typescript
 const trackId = '7ouMYWpwJ422jRcDASZB7P'; 
+const audioAnalysis = await spotified.track.getTracksAudioAnalysis(trackId);
+console.log(audioAnalysis);
+```
+
+### getRecommendations(seedParams: RecommendationSeedParams, optionalParams: RecommendationOptionalParams)
+
+This method is used to get recommendations are generated based on the available information for a given seed entity and matched against similar artists and tracks.
```

Comment:
Grammaticaly error after recommendations. Replace the whole line with the text below

This method is used to get recommendations that are generated based on the available information for a given seed entity and matched against similar artists and tracks.